src/Pages/Books.jsx

```diff
@@ -1,27 +1,18 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Book from '../Components/Book';
 import InputData from '../Components/InputData';
+import { deleteBook } from '../redux/books/books';
 
 function Books() {
-  const booksList = [{
-    title: 'test1',
-    author: 'test1',
-  }, {
-    title: 'test2',
-    author: 'test2',
-  }, {
-    title: 'test3',
-    author: 'test3',
-  }, {
-    title: 'test4',
-    author: 'test4',
-  }];
+  const books = useSelector((state) => state.book);
+  const dispatch = useDispatch();
   return (
     <>
-      {booksList.map((book) => (
-        <div key={book.title}>
+      {books.map((book) => (
+        <div key={book.id}>
           <Book title={book.title} author={book.author} />
-          <button type="button">Remove</button>
+          <button type="button" onClick={() => { dispatch(deleteBook(book.id)); }}>Remove</button>
```

Comment:
- [ ] Once more let's kindly take a look at the requirements. The button for removing the book should use the function `removeBookFromStore`(Hi @ichala don't forget that you can name it differently). This function should be made in this component(`Books`). The function takes the id from the book data, and dispatches the `deleteBook` action creator with the id as its argument :+1: